site/en/r2/tutorials/eager/custom_training.ipynb

```diff
@@ -142,9 +142,9 @@
         "id": "wfneTXy7JcUz"
       },
       "source": [
-        "TensorFlow, however, has stateful operations built in, and these are often more pleasant to use than low-level Python representations of your state. To represent weights in a model, for example, it's often convenient and efficient to use TensorFlow variables.\n",
+        "TensorFlow, however, has stateful operations built in, and these are often more pleasant to use than low-level Python representations of your state. To represent weights in a model, for example, it's often convenient and efficient to use TensorFlow variables, which are manipulated via the `tf.Variable` class.\n",
```

Comment:
"it's often convenient and efficient to use TensorFlow variables, which are manipulated via the `tf.Variable` class"


How about just:

"To represent weights in a model use the `tf.Variable` class."

Reply:
Agreed, @MarkDaoust. Done. I should watch the [style and tone](https://developers.google.com/style/tone).